disco/api/http.py

```diff
@@ -189,7 +189,7 @@ class HTTPClient(LoggingClass):
     A simple HTTP client which wraps the requests library, adding support for
     Discords rate-limit headers, authorization, and request/response validation.
     """
-    BASE_URL = 'https://discordapp.com/api/v7'
+    BASE_URL = 'https://discord.com/api/v6'
```

Comment:
For now lets leave this as v7. I've confirmed with Discord staff that they don't currently intend on removing this version (which introduced better response payloads for errors) despite its lack of "official" support.

Reply:
right; there was no mention of it in their docs. Much clandestine such fun.

```suggestion
    BASE_URL = 'https://discord.com/api/v7'
```